### PR TITLE
MB-8203 show existing agents

### PR DESCRIPTION
**Pull request by mlm55**
## Description

Fixes a bug where existing agents weren't shown on the shipment edit form

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make office_client_run
make server_run
```
I think you should be able to hit this website and see the form here: http://officelocal:3000/counseling/moves/SRVCSL/3cc2d807-2905-4ff5-9ce6-147c101652b4/edit
Neither of the shipments have agents yet, so edit one of them and add a shipment. Then try to edit the shipment again, and the agent fields should be filled in.

## Code Review Verification Steps
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8203) for this change

### CODE REVIEW

File: src/components/Office/ServicesCounselingShipmentForm/ServicesCounselingShipmentForm.jsx [hunk 1]
```diff
@@ -52,9 +52,9 @@ const ServicesCounselingShipmentForm = ({
   const { showDeliveryFields, showPickupFields, schema } = getShipmentOptions(shipmentType);
   const isNTS = shipmentType === SHIPMENT_OPTIONS.NTS;
   const shipmentNumber = shipmentType === SHIPMENT_OPTIONS.HHG ? getShipmentNumber() : null;
-
-  const initialValues = formatMtoShipmentForDisplay(isCreatePage ? {} : mtoShipment);
-
+  const initialValues = formatMtoShipmentForDisplay(
```
Comment (YanZ777) [inline src/components/Office/ServicesCounselingShipmentForm/ServicesCounselingShipmentForm.jsx:55]: Tests please!

**Comment by YanZ777** [comment]
It looks like your description is linked to the wrong Jira issue.

**Review by felipe-lee — COMMENTED** [review]
This works for me! But I agree with Yan, can you add tests that will ensure it shows up as expected going forward.

File: src/components/Office/ServicesCounselingShipmentForm/ServicesCounselingShipmentForm.jsx (hunk 1 above)
Comment (mlm55) [reply src/components/Office/ServicesCounselingShipmentForm/ServicesCounselingShipmentForm.jsx:55]: added!

**Review by felipe-lee — APPROVED** [review]
LGTM, thanks for adding tests!

**Comment by felipe-lee** [comment]
Btw, you'll need to merge master into this branch to get past the CI/CD errors. There's an outdated dependency that's been fixed in master.

**Review by YanZ777 — APPROVED** [review]
Thanks for adding those tests!

**Pull request merged**